src/components/Search/SearchRouter/SearchRouter.tsx

```diff
@@ -70,11 +125,21 @@ function SearchRouter() {
             onClose={closeSearchRouter}
         >
             <FocusTrapForModal active={isSearchRouterDisplayed}>
-                <View style={[styles.flex1, styles.p3]}>
+                <View style={[styles.flex1, styles.p3, modalWidth, styles.mh100, !isFullScreen && styles.mh85vh]}>
                     <SearchRouterInput
-                        isFullWidth={isFullWidth}
                         onChange={onSearchChange}
-                        onSubmit={onSearchSubmit}
+                        onSubmit={() => {
+                            onSearchSubmit(currentQuery);
+                        }}
+                    />
+
+                    <SearchRouterList
+                        currentSearch={currentQuery}
```

Comment:
can we do:
```suggestion
                      currentQuery={currentQuery}
```

to have less different names?